src/Uno.UI.Runtime.Skia.Gtk/Rendering/GLRenderSurfaceBase.cs

```diff
@@ -190,6 +192,10 @@ public void TakeScreenshot(string filePath)
 			}
 		}
 
-		private void UpdateDpi() => _scale = (float)_displayInformation.RawPixelsPerViewPixel;
+		private void UpdateDpi()
+		{
+			_scale = (float)_displayInformation.RawPixelsPerViewPixel;
```

Comment:
Let's explain why we're doing this (because native controls don't support fractions)

Reply:
Added this explainer to the new display information property